components/root/root.jsx

```diff
@@ -189,6 +189,14 @@ export default class Root extends React.Component {
             }
 
             this.setState({configLoaded: true});
+
+            const enableDeveloper = getConfig(store.getState()).EnableDeveloper;
```

Comment:
As you've discovered, our config unfortunately passes down strings for these settings. However, we try to push that up as high as possible to as to use booleans where it matters. Can you move the `=== 'true'` check into this line instead?